frontend/src/app/home/homepage.component.scss

```diff
@@ -1,6 +1,6 @@
-volontulo-homepage-offer {
-    margin-bottom: 40px;
-}
+@import "../../sass/variables";
```

Comment:
I spoke with @Bosmanfrx and it looks that these two imports are not necessery.

We should not bloat our styles so, please, remove them.

Reply:
ok, removed.